section-engineering-education/engineering-education
# Adding Individual Author RSS Feeds

### What This PR Does

- Adds a term in category RSS Feed template for individual author RSS feeds so readers can subscribe to an author's post via an RSS service (such as Feedly)
- Changes the author profile files to be a taxonomy in addition to a section - only `louise-findlay` and `ahmad-mardeni` as a test

### Files Edited

- **Authors Taxonomy Term RSS Feed Template** - `/layouts/authors/rss.xml`
- **Louise Findlay's Author Profile** - `/content/authors/louise-findlay/index.md`
- **Ahmad Mardeni's Author Profile** - `/content/authors/ahmad-mardeni/index.md`

### How to Test

1. Checkout this PR branch
2. Start the local hugo server - `hugo server -D`
3. Navigate to [localhost:1313/authors/louise-findlay/index.xml](http://localhost:1313/authors/louise-findlay/index.xml)
4. Verify that the RSS Feed shown matches the screenshot below and that the Author Profile still looks as previously
5. Navigate to [localhost:1313/authors/ahmad-mardeni/index.xml](http://localhost:1313/authors/ahmad-mardeni/index.xml) and verify the articles listed are only from this author

### Screenshot

![Ahmad Mardeni's RSS Feed](https://user-images.githubusercontent.com/26024131/158065005-678c22ef-ece2-4b9a-a276-455d65265202.png)

### Future Steps

- Add `authors: {author-foldername}` to the front-matter of all other author profiles to display the feeds for other authors

### Related Issue

This closes #7105.
